$OpenBSD: patch-man_Makefile,v 1.1.1.1 2012/05/02 07:37:10 jasper Exp $

Allow overriding of a2x in the environment; as it may also be installed as a2x.py (committed upstream).


--- man/Makefile.orig	Fri Apr 27 16:00:41 2012
+++ man/Makefile	Sun Apr 29 15:03:11 2012
@@ -1,6 +1,8 @@
 all: i3status.1
 
+A2X?=a2x
+
 i3status.1: asciidoc.conf i3status.man
-	a2x -f manpage --asciidoc-opts="-f asciidoc.conf" i3status.man
+	${A2X} -f manpage --asciidoc-opts="-f asciidoc.conf" i3status.man
 clean:
 	rm -f i3status.xml i3status.1 i3status.html
